providers.py

```diff
@@ -18,3 +18,21 @@ def post_google(data):
     )
     json_response = json.loads(response.text)
     return json_response
+
+
+def get_categories_from_cms():
+    if 'CMS_URL' not in os.environ:
+        return []
+
+    response = requests.get(f'{os.environ["CMS_URL"]}/categories.json?cache=false')
+    json_response = json.loads(response.text)
+    return json_response['data']
```

Comment:
These methods make the API call to the CMS to get JSON for the categories and the states. The CMS has not been deployed yet, so I'm having this return an empty array (e.g. do nothing) if the CMS URL is not in the environment variables. I'll add it to the environment variables in Heroku once it's launched.